src/collections/_documentation/server/sso.md

```diff
@@ -8,54 +8,25 @@ SSO in Sentry is handled in one of two ways:
 -   Via a middleware which handles an upstream proxy dictating the authenticated user
 -   Via a third-party service which implements an authentication pipeline
 
-This documentation describes the latter, which would cover things like Google Apps, GitHub, LDAP, and other similar services.
+This documentation describes the latter, which would cover things like Google Apps and GitHub.
 
-## Enabling SSO
-
-As of version 8.0 the SSO feature is enabled by default in Sentry. That said it can be disabled with a feature switch in your `sentry.conf.py`:
-
-```python
-from sentry.conf.server import *
-
-# turn SSO on our off
-SENTRY_FEATURES['organizations:sso'] = False
-```
-
-Additionally you may enable advanced SSO features:
-
-```python
-from sentry.conf.server import *
-
-SENTRY_FEATURES['organizations:sso-saml2'] = True
```

Comment:
@BYK curious why we are removing the example of Saml2 and rippling?

Reply:
This is not an example, it is just showing a feature flag that is obsolete as it is enabled by default for a while now.

Reply:
@EvanPurkhiser it is obsolete for on-premise as it is enabled by default there.